spec/v1.0/resource_descriptor.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package github.intoto.attestation.spec.v1_0;
+
+import "google/protobuf/struct.proto";
+
+option go_package = "github.com/in-toto/attestation/go/spec/v1.0";
```

Comment:
From what I can tell the go package does double-duty.  The convention seems to be that it's just a single word here (we'd have `attestation`) or something like that, but then users would still use github.com/... to import it in their code because that's how they tell Go where to find it?

I can try making that change, but it would be larger and should probably be in a separate PR.

Reply:
I just went to see how others do it, and you're right that the `go_package` should point to the download location of the .go files, so we probably keep what we have.

Reply:
Yep, agreed!